app/src/androidTest/java/org/mozilla/tv/firefox/helpers/FakePocketVideoRepoProvider.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import org.mozilla.tv.firefox.ext.toUri
+import org.mozilla.tv.firefox.pocket.PocketEndpoint
+import org.mozilla.tv.firefox.pocket.PocketFeedStateMachine
+import org.mozilla.tv.firefox.pocket.PocketVideoRepo
+import org.mozilla.tv.firefox.pocket.PocketViewModel
+import org.mozilla.tv.firefox.utils.BuildConfigDerivables
+
+/**
+ * Provides a fake [PocketVideoRepo] implementation for testing purposes.
+ *
+ * Any values pushed to [fakedPocketRepoState] will be immediately emitted.
+ */
+object FakePocketVideoRepoProvider {
```

Comment:
Out of scope of this PR but I don't love that this provider is not created with composable parts: i.e. both tests rely on this code but if they needed to change it (e.g. adding functionality), this class would need to change so both tests are affected by those changes (they're coupled together).

I haven't given enough thought about how to fix this but we can do it later as we write more tests, or run into this problem for real.